src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1038,6 +1039,32 @@ function ($a) {
                                 ->end()
                             ->end()
                         ->end()
+                        ->scalarNode('default_bus')->defaultValue('default')->end()
+                        ->arrayNode('buses')
+                            ->defaultValue(array('default' => array('middlewares' => $defaultMiddlewares = array(array('service' => 'logging'), array('service' => 'route_messages'), array('service' => 'call_message_handler')))))
+                            ->useAttributeAsKey('name')
+                            ->prototype('array')
+                                ->fixXmlConfig('middleware')
+                                ->addDefaultsIfNotSet()
+                                ->children()
+                                    ->arrayNode('middlewares')
+                                        ->defaultValue($defaultMiddlewares)
+                                        ->prototype('array')
+                                            ->beforeNormalization()
+                                                ->ifString()
+                                                ->then(function (string $middleware) {
+                                                    return array('service' => $middleware);
+                                                })
+                                            ->end()
+                                            ->children()
+                                                ->scalarNode('service')->isRequired()->end()
+                                                ->integerNode('priority')->defaultValue(0)->end()
```

Comment:
can be removed now? 😄The `service` key wouldn't be necessary anymore neither so it can be simplified.